src/dev/flang/fe/SourceModule.java

```diff
@@ -181,7 +181,15 @@ String parseMain()
                 f.legalPartOfUniverse();  // suppress FeErrors.initialValueNotAllowed
                 if (stmnts.size() == 1)
                   {
-                    res =  f.featureName().baseName();
+                    if (f.kind() == Feature.Kind.Field)
+                      {
+                        // execute the universe's code in this case, to avoid FeErrors.mainFeatureMustNotBeField
+                        res = null;
```

Comment:
would be better to keep `res` unchanged in this case, i.e., return `_defaultMain`. The comment `@return...` should also be fixed to reflect that this returns `_defaultMain` (even if, as far as I can see, this should be `null`.